packages/playground/src/utils/get_nodes.ts

```diff
@@ -92,11 +92,13 @@ export const getNodeStatusColor = (status: string): NodeStatusColor => {
   }
 };
 
-export const getNodeTypeColor = (dedicated: boolean): NodeTypeColor => {
-  if (dedicated) {
-    return { color: "primary", type: "dedicated" };
-  } else {
+export const getNodeTypeColor = (dedicated: boolean, rentedByTwinId: number): NodeTypeColor => {
+  if (!dedicated) {
     return { color: "success", type: "shared" };
+  } else if (dedicated && rentedByTwinId === 0) {
```

Comment:
No need for else ifs you already returned in the previous block 
if !dedicated {
   return xyz 
   }
   
  // here it's dedicated
  if dedicated && ... {
  
  } else {
     
  }